docs/src/tutorials/applications/web_app.jl

```diff
@@ -0,0 +1,141 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Serving web apps
+
+# This tutorial demonstrates how to setup and serve JuMP models via a REST API.
+
+# First, we need the standard JuMP packages:
```

Comment:
Is HiGHS a standard JuMP package? I also believe that beginners may be a bit puzzled. I'd rather say "we need JuMP and a solver".